src/dndbeyond/content-scripts/character.js

```diff
@@ -513,8 +514,7 @@ function handleSpecialWeaponAttacks(damages=[], damage_types=[], properties, set
 
     if (character.hasClass("Paladin")) {
         // Paladin: Sacred Weapon
-        if (to_hit !== null && 
-            character.getSetting("paladin-sacred-weapon", false)) {
+        if (character.getSetting("paladin-sacred-weapon", false)) {
```

Comment:
same question about removing check for to_hit

Reply:
There is no condition under which there isn't a `to_hit` on this now in the function call